DisplayCAL/display_cal.py

```diff
@@ -18147,11 +18147,11 @@ def load_cal_handler(
                     if 'HIRES_B2A "YES"' not in ti3_lines:
                         setcfg("profile.b2a.hires", 0)
                     setcfg("profile.b2a.hires.smooth", 0)
+                simset = False  # Only HDR 3D LUTs will have this set
```

Comment:
simset has to be set to False even if "BEGIN_DATA_FORMAT" was not found in ti3. Is that intentional?

Reply:
intentional, I noticed it blowing during testing and then I can see my IDE flags it 

![image](https://github.com/eoyilmaz/displaycal-py3/assets/2487509/1df26cd9-5bb0-49f4-9008-8d354944831c)